### PR TITLE
Display DMT on README.md

### PR DESCRIPTION
**Pull request by ishmum123**
As requested by @zhaber

### CODE REVIEW

**Comment by zhaber** [comment]
For an end-to-end BigQuery migration solution, use [Data Migration Tool (DMT)](https://github.com/GoogleCloudPlatform/data-migration-tool). DMT currently supports Redshift, TeraData, Hive, and Oracle as data sources and BigQuery as the destination.

**Review by shevek-google — CHANGES_REQUESTED** [review]
Could the DMT group please give us a demo before we approve this PR?

**Comment by zhaber** [comment]
Added DMT demo to our Nov 1 meeting agenda

**Comment by shevek-google** [comment]
> Added DMT demo to our Nov 1 meeting agenda

Could you invite me to that meeting, please? I don't see it. Also, can we please do a day when I'm in the office? I'm out on 1st Nov.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 This project contains a collection of tools related to the [BigQuery Migration
 Service](https://cloud.google.com/bigquery/docs/migration-intro).
 
+For an end-to-end BigQuery migration solution, use [Data Migration Tool (DMT)](https://github.com/GoogleCloudPlatform/data-migration-tool). 
+DMT currently supports Redshift, TeraData, Hive, and Oracle as data sources and BigQuery as the destination.
```
Comment (dawidxc) [inline README.md:8]: nit: Teradata

**Comment by ishmum123** [comment]
@zhaber any updates on this?

**Comment by ishmum123** [comment]
Closing due to no action

**Comment by zhaber** [comment]
I believe @shevek-google is following up on this

**Pull request closed without merging**